content/applications/hr/fleet/service.rst

```diff
@@ -86,61 +298,219 @@ clicked, sort the data in different ways.
 
-The information can be sorted by any column. At the top of each column, hover over the column name.
-An arrow appears to the right of the name. Click the arrow to sort the data by that specific column.
+The two exceptions to this sorting are the default :guilabel:`Date` column and the :guilabel:`Cost`
+column. The :guilabel:`Date` column sorts the information in chronological order (January to
+December) instead of alphabetical order, while the :guilabel:`Cost` column sorts the information by
+repair price, from lowest to highest.
 
-The default sorting is descending alphabetical order (A to Z). Click the arrow again to reverse the
-order, and go in reverse alphabetical order (Z to A). The one exception to this sorting is the
-default :guilabel:`Date` column, which sorts the information in chronological order (January to
-December) instead of alphabetical order.
+Add a service
+*************
+
+To add a service record from the list view, click the :guilabel:`New` button, and a service form
+loads. :ref:`Enter all the information <service/service-form>` on the service form.
+
+The form automatically saves as data is entered.
 
 Kanban view
 ~~~~~~~~~~~
 
 To view services by their stage, click on the :guilabel:`Kanban icon`, which is the second icon in
-the top right, and appears as four black squares in a cube.
+the top-right corner, and appears as two different length bars beneath a line.
+
+All services with the same status appear in the corresponding Kanban column, from
+:guilabel:`Running` to :guilabel:`Canceled`.
+
+The number of repairs for each type of service appears in the far-right of each Kanban column
+header.
+
+The status of any scheduled activities appears in the color-coded bar beneath each Kanban column
+title. Repairs with activities scheduled in the future appear green, activities due today appear
+yellow, overdue activities appear red, and repairs with no activities scheduled appear gray.
+
+Each Kanban card displays a color-coded :guilabel:`(activity icon)`, indicating what type of
+activity is scheduled, and the status. For example, a scheduled phone call displays a phone icon.
+The status colors correspond to the colors in the status bar. Click on an :guilabel:`(activity
+icon)` to view the details of the activity.
+
+The length of the color bar is proportionate to the number of corresponding activities with that
+status. Hover over a color section to reveal the number of service records in that specific
+grouping.
+
+.. image:: service/kanban.png
+   :align: center
+   :alt: The service records presented in a Kanban view.
+
+To view **only** the records with a specific status, click the desired :guilabel:`color bar`
+section. The background color for the column changes to a pale hue of the same color (either green,
+yellow, red, or gray), and the color bar appears striped instead of solid. **Only** repairs and
+services with the selected status appear in the column.
```

Comment:
I put a gui on that word originally because you do see the bar on the UI and you can click it- but unlike the Kanban button, this doesn't actually say "color bar" when you hover over it, so I agree. As for it being too granular- I see your point, but I also see the value in having it. My reasoning is, this info I do not see in other places. There is an activities doc now in the new Odoo Essentials (I am not sure if that was there when I first wrote this doc- I can't remember, but that is a recent addition). The issue is- that doc doesn't explain what the color bar means or is (and I'll admit, it took me a min to figure it out since in runbot it's always gray, and I needed to add a bunch of activities to see what happens. I didn't realize you could click on a color and show ONLY those entries as well!). So since it seemed like a feature for me, I figured I couldn't be the only one out there who would find it valuable.